packages/interaction/src/InteractionManager.js

```diff
@@ -647,6 +645,7 @@ export default class InteractionManager extends EventEmitter
          * @event PIXI.DisplayObject#touchmove
          * @param {PIXI.interaction.InteractionEvent} event - Interaction event
          */
+        this.setTargetElement(this.renderer.view, this.renderer.resolution);
```

Comment:
style wise i'd move this just 1 line lower with a blank space in between, otherwise it looks like you're documenting setTargetElement